src/luaotfload-colors.lua

```diff
@@ -177,7 +177,7 @@ local function node_colorize (head, toplevel, current_color, current_transparent
     while n do
         local n_id = getid(n)
 
-        if n_id == hlist_t or n_id == vlist_t then
```

Comment:
This is necessary when dealing with nodes generated from math mode. E.g. `\[ \frac{a}{b} \]` would no longer color `a` and `b` after this change.